tox.ini

```diff
@@ -21,5 +21,5 @@ deps =
     django-110: Django>=1.10,<1.11
 commands =
     flake8: flake8 safedelete --ignore=E501
-    django: coverage run --parallel-mode {toxinidir}/runtests.py
```

Comment:
Forgot to mention this in the PR. I added the posargs variable in the tox setup, [more info](http://tox.readthedocs.io/en/latest/example/general.html).